tests/test_abc.py

```diff
@@ -39,11 +39,14 @@ def __init__(
         *,
         cache=None,
         oauth_token=None,
+        base_url=None,
```

Comment:
Hmm, should this also be defaulted to sansio.DOMAIN instead of None?

Reply:
Good catch. I think it should change as well. Might as well just type all of this as `str` and not worry about `None`.